components/crud-web-apps/volumes/frontend/src/app/pages/form/form-default/form-default.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="form--container" mat-dialog-content>
+  <form novalidate (ngSubmit)="onSubmit()" [formGroup]="formCtrl">
+    <lib-form-section
+      title="New Volume"
+      text="Create a new empty Volume or clone an existing snapshot."
```

Comment:
I don't see an option to allow cloning from a snapshot in the UI, while it is listed here.

Reply:
Will need rephrase this to remove the reference for snapshots from the default flavor of the app

Reply:
pushed a commit to rephrase this text